docs/units-and-global-variables.rst

```diff
@@ -180,6 +180,10 @@ For more information, see the section on :ref:`address`.
     to make safe Ether transfers, always check the return value of ``send``, use ``transfer`` or even better:
     Use a pattern where the recipient withdraws the money.
 
+.. note::
+   Before version 0.5.0 Solidity allowed address members to be accessed by a contract instance, for example ``this.balance``.
+   This is now forbidden and an explicit convertion to address must be done: ``address(this).balance``.
```

Comment:
There's a typo: "convertion". Should we either pre-format "address" with back-ticks or use it as a hyperlink to http://solidity.readthedocs.io/en/v0.4.24/types.html#address, as it is done in the note section above?

Reply:
Regarding address, I think it's fine this way since there's already a general `For more information, see the section on :ref:address.` above.